packages/formation-react/src/components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const Sidebar = props => (
+  <div className="medium-screen:vads-u-padding-left--4">{props.children}</div>
+);
+
+Sidebar.propTypes = {
+  classes: PropTypes.string,
+};
```

Comment:
I think we should use something other than `Sidebar` as the name - either something to express that it's usually used as the _right_ sidebar (eg `RightRail`) or possibly to describe the way it's used - maybe `FaqSidebar` or something like that. `Sidebar` currently is used basically interchangeably with the [left nav](https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/platform/site-wide/side-nav).

Whatever name we choose, it will probably end up changed it before it is formalized and published to design.va.gov. So for now, I think we should shoot for a name that descriptive enough (possibly overly descriptive) to make it clear to devs as the audience.

Reply:
I agree, I renamed this <RightRail /> and the blocks to <RightRailBlock />